content/manuals/desktop/use-desktop/images.md

```diff
@@ -87,10 +87,10 @@ To remove individual images, select the bin icon.
 
 The **Images** view also allows you to manage and interact with images in Docker Hub repositories.
 By default, when you go to **Images** in Docker Desktop, you see a list of images that exist in your local image store.
-The **Local** and **Hub** tabs near the top toggles between viewing images in your local image store,
+The **Local** and **Docker Hub** tabs near the top toggles between viewing images in your local image store,
```

Comment:
We renamed this one to 'Hub repositories'. Sorry it is a bit confusing. Basically the new tab screen on the left navigation panel is 'Docker Hub' and the inner tab in the Images screen is now 'Hub repositories'.

![Screenshot 2024-12-06 at 11 47 31 AM](https://github.com/user-attachments/assets/f9b4c818-88be-42b0-b371-e96ff443520f)